web/modules/custom/batch_update/batch_update.post_update.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Implements hook_post_update_NAME.
+ */
+
+/**
+ * Get paragraph and call function for update text format.
+ */
+function batch_update_post_update_format3(&$context): void {
+  $paragraph = \Drupal::entityQuery('paragraph');
+  $paragraph->accessCheck(FALSE)
+    ->exists('field_regular_text');
+  $paragraphs = $paragraph->execute();
+  $limit = 50;
```

Comment:
you could also avoid querying all of IDs
To make sure that this will not fault no matter of how much entities you have.